airflow/www/views.py

```diff
@@ -702,6 +703,25 @@ def _iter_parsed_moved_data_table_names():
                 # Second segment is a version marker that we don't need to show.
                 yield segments[2], table_name
 
+        robots_file_access_count = session.query(Log).filter(Log.event == "robots").filter(
```

Comment:
This query should only be run if `webserver.warn_deployment_exposure` is enabled. Otherwise its putting additional load on the database for no reason, since the warning isn't going to be displayed anyways.